poller/poller_test.go

```diff
@@ -48,6 +49,11 @@ func (splitio *mockSplitio) GetSegmentsForSplits(splits map[string]dtos.SplitDTO
 	return nil, 0, fmt.Errorf("Error from splitio API when getting segments")
 }
 
+// getSplitData helper returns split data cache results
+func getSplitData(poller *Poller) SplitData {
```

Comment:
So @rohanmurthy will refactor this right?

Reply:
yep! he'll use the new private function instead